feature/aft/aft_base/otg_tests/aft_base/README.md

```diff
@@ -0,0 +1,140 @@
+# AFT-1.1: AFTs Base
+
+## Summary
+
+IPv4/IPv6 unicast routes next hop group and next hop.
+
+## Procedure
+
+Configure DUT:port1,port2,port3 for IS-IS session with ATE:port1,port2,port3
+*   IS-IS must be level 2 only with wide metric.
+*   IS-IS must be point to point.
+*   Send 1000 ipv4 and 1000 ipv6 IS-IS prefixes from ATE:port3 to DUT:port3.
+*   IS-IS must be up and running, prefixes must be in RIB and FIB.
```

Comment:
What is the proposed method of confirming the values are in the RIB and FIB using the provided paths?

(Same question in the other README.)

Reply:
The README should be precise in how the validation is performed.  (if the go test code was included, we could use the code to review the most specific options.  You are welcome to include the code and the readme in the same PR).

For example, we use gnmi subscribe with ON_CHANGE option.  So the subscription should start before advertising prefixes.  

```suggestion
* Use gnmi Subscribe with ON_CHANGE option to /network-instances/network-instance/afts 

* Advertise 1,000 ipv4,ipv6 prefixes from ATE port1, port2 
* Verify afts prefix entries using the folllowing paths within a timeout period of 10 seconds

```
/network-instances/network-instance/afts/ipv4-unicast/ipv4-entry/state/prefix
/network-instances/network-instance/afts/ipv4-unicast/ipv4-entry/state/origin-protocol
/network-instances/network-instance/afts/ipv6-unicast/ipv6-entry/state/prefix
/network-instances/network-instance/afts/ipv6-unicast/ipv6-entry/state/origin-protocol
/network-instances/network-instance/afts/ipv6-unicast/ipv6-entry/state/counters
```

```

Reply:
done

Reply:
Hi, I don't see the recommended paths to be verified in this section as requested?

```
/network-instances/network-instance/afts/ipv4-unicast/ipv4-entry/state/prefix
/network-instances/network-instance/afts/ipv4-unicast/ipv4-entry/state/origin-protocol
/network-instances/network-instance/afts/ipv6-unicast/ipv6-entry/state/prefix
/network-instances/network-instance/afts/ipv6-unicast/ipv6-entry/state/origin-protocol
```

Reply:
added